webview.hpp

```diff
@@ -452,8 +452,14 @@ int WebView::init() {
                  r.bottom - r.top,
                  SWP_NOZORDER | SWP_NOACTIVATE | SWP_FRAMECHANGED);
 
+    if (!resizable) {
+       auto style = GetWindowLong(hwnd, GWL_STYLE);
+       style &= ~(WS_THICKFRAME | WS_MAXIMIZEBOX);
+       SetWindowLong(hwnd, GWL_STYLE, style);    
+    } else {
     // Used with GetWindowLongPtr
-    SetWindowLongPtr(hwnd, GWLP_USERDATA, (LONG_PTR)this);
+      SetWindowLongPtr(hwnd, GWLP_USERDATA, (LONG_PTR)this);
+    };
```

Comment:
Works great, thanks for the change! Some minor things:
- I think this should use `GetWindowLongPtr` and `SetWindowLongPtr` instead of `GetWindowLong` and `SetWindowLong` as the `Ptr` versions work with [both 32-bit and 64-bit Windows](https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowlongptra)
- The `SetWindowLongPtr` in the `else` statement is related to something else ([over here](https://github.com/MichaelKim/webview/blob/66697937305b6cc78e75df5e4afc2cf0de8b24a2/webview.hpp#L632)), so it shouldn't be part of it.
- If you'd like, you could apply this fix to the old Edge too ([over here](https://github.com/MichaelKim/webview/blob/66697937305b6cc78e75df5e4afc2cf0de8b24a2/webview.hpp#L251)).

Perhaps something like:
```c++
if (!resizable) {
    auto style = GetWindowLongPtr(hwnd, GWL_STYLE);
    style &= ~(WS_THICKFRAME | WS_MAXIMIZEBOX);
    SetWindowLongPtr(hwnd, GWL_STYLE, style);
}

// Used with GetWindowLongPtr in WndProcedure
SetWindowLongPtr(hwnd, GWLP_USERDATA, (LONG_PTR)this);
```